package.json

```diff
@@ -2,24 +2,30 @@
   "name": "ml-spectra-processing",
   "version": "6.8.0",
   "description": "Various method to process spectra",
-  "main": "lib/index.js",
-  "module": "src/index.js",
+  "main": "./lib/index.js",
+  "module": "./lib-esm/index.js",
+  "types": "./lib/index.d.ts",
   "files": [
+    "src",
     "lib",
-    "src"
+    "lib-esm"
   ],
   "scripts": {
     "build": "rollup -c && cheminfo-build --root SpectraProcessing",
-    "compile": "rollup -c",
+    "check-types": "tsc --noEmit",
+    "clean": "rimraf lib lib-esm",
     "eslint": "eslint src",
     "eslint-fix": "npm run eslint -- --fix",
     "jscpd": "jscpd -l 10 -i \"**/__tests__/**\" -t 1 src",
-    "prepack": "npm run compile",
+    "prepack": "npm run compile && npm run tsc",
```

Comment:
You should keep only `npm run tsc`. compile script was removed

Reply:
Resolved